docs/explanation/core/proposer-based-timestamps.md

```diff
@@ -55,44 +33,182 @@ The following protocols and application features require a reliable source of ti
 
-### Proposer Selects a Block Timestamp
+### Enabling PBTS
+
+The PBTS algorithm is **not enabled by default** in CometBFT v1.0.
+
+If a network upgrades to CometBFT v1.0, it will still use the BFT Time
+algorithm until PBTS is enabled.
+The same applies to new networks that do not change the default values for
+consensus parameters in the genesis file.
+
+Enabling PBTS requires configuring the [consensus parameters](#consensus-parameters)
+that govern the operation of the algorithm.
+
+### Selecting a Timestamp
 
-When the proposer node creates a new block proposal, the node reads the time
-from its local clock and uses this reading as the timestamp for the proposed
-block.
+When a validator creates a new block, it reads the time from its local clock
+and uses this reading as the timestamp for the block.
+The proposer of a block is thus free to select the block timestamp, but this
+timestamp has to be validated by other nodes in the network.
 
-### Timeliness
+### Validating Timestamps
 
-When each validator on the network receives a proposed block, it
-performs a series of checks to ensure that the block can be considered valid as
-a candidate to be the next block in the chain.
+When each validator on the network receives a proposed block, it performs a
+series of checks to ensure that the block can be considered valid as a
+candidate to be the next block in the chain.
+If the block is considered invalid, the validator issues a `nil` prevote,
+signaling to the rest of the network that the proposed block is not valid.
 
 The PBTS algorithm performs a validity check on the timestamp of proposed
 blocks. When a validator receives a proposal it ensures that the timestamp in
-the proposal is within a bound of the validator's local clock. Specifically, the
-algorithm checks that the timestamp is no more than `Precision` greater than the
-node's local clock and no less than `Precision + MessageDelay` behind than the
-node's local clock. This creates range of acceptable timestamps around the
-node's local time. If the timestamp is within this range, the PBTS algorithm
-considers the block **timely**. If a block is not **timely**, the node will
-issue a `nil` prevote for this block, signaling to the rest of the network
-that the node does not consider the block to be valid.
+the proposal is within a bound of the validator's local clock.
+For that it uses `Precision` and `MessageDelay` consensus parameters, 
+ which are the same across all nodes.
+Specifically, the algorithm checks that the timestamp is
+no more than `Precision` greater than the node's local clock
+(i.e., not in the future)
+and no less than `MessageDelay + Precision` behind the node's local clock
+(i.e., not too far in the past).
+If the proposed block's timestamp is within the window of acceptable
+timestamps, the timestamp is considered **timely**.
+If the block timestamp is **not timely**, the validator rejects the block by
+issuing a `nil` prevote.
 
```

Comment:
Should we add that this check is not performed if the proposed block has been locked?

Reply:
In this case, the proposed block is the same, but it is re-proposed in a different round. Also, this may be too much detail for a doc.

Reply:
Added this observation in https://github.com/cometbft/cometbft/pull/2376/commits/915fda848a656a7da36ca248f21cde10da03fae7